src/proc/namer.rs

```diff
@@ -57,7 +57,14 @@ impl Namer {
 
     fn call_or(&mut self, label: &Option<String>, fallback: &str) -> String {
         self.call(match *label {
-            Some(ref name) => name,
+            Some(ref name) => {
+                // Name can be empty for some cases in SPIR-V
```

Comment:
there is only one case where this happens in practice, and `spv` frontend should be handling it, not `Namer`

Reply:
One more try. Thank you for your patience :smile: